testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java

```diff
@@ -44,4 +59,16 @@ public static boolean updatePassword(RealmResource realm, String username, Strin
         }
     }
 
+    public static String getEmail(RealmResource realm, String username) {
```

Comment:
I think these methods are too fine-grained for a generic `AccountHelper` class. Each of them is calling `getUserRepresentation`, which performs a search query. If you need email, first name and last name in a test, you are querying the DB 3 times. 

I think it's better to use `getUserRepresentation` directly in the test method.

Reply:
There are 14 usages of getEmail(), and 15 of getFirstName() and getLastName(). It is for reusability of the code and easy the modification of it